docs/_docs/containers-ascollectionnode.md

```diff
@@ -225,4 +225,4 @@ The most detailed example of laying out the cells of an `ASCollectionNode` is th
 
 Note that these UIKit cells will **not** have the performance benefits of `ASCellNodes` (like preloading, async layout, and async drawing), even when mixed within the same `ASCollectionNode`. 
 
-However, this interoperability allows developers the flexibility to test out the framework without needing to convert all of their cells at once. Read more <a href="uicollectionviewinterop.html">here</a>.
\ No newline at end of file
+However, this interoperability allows developers the flexibility to test out the framework without needing to convert all of their cells at once. Read more <a href="uicollectionviewinterop.html">here</a>.
```

Comment:
What changed here? I feel like I'm taking crazy pills! 🤪

Reply:
Ah, think that was Xcode freaking out at a missing new line at the end.